NGitLab/PublicAPI.Unshipped.txt

```diff
@@ -2773,22 +2748,22 @@ NGitLab.Models.User.CanCreateGroup -> bool
 NGitLab.Models.User.CanCreateProject -> bool
 NGitLab.Models.User.ColorSchemeId -> int
 NGitLab.Models.User.CommitEmail -> string
-NGitLab.Models.User.ConfirmedAt -> System.DateTime
+NGitLab.Models.User.ConfirmedAt -> System.DateTime?
 NGitLab.Models.User.CreatedAt -> System.DateTime
-NGitLab.Models.User.CurrentSignIn -> System.DateTime
+NGitLab.Models.User.CurrentSignIn -> System.DateTime?
```

Comment:
Could we implement a custom converter to set the prop to `DateTime.MinValue` (current behavior) when the json prop is null?